installer/conf/omsagent.d/npmd.conf

```diff
@@ -1,9 +1,9 @@
 <source>
     type npmd
     tag oms.npmd
-    location_unix_endpoint  "/var/opt/microsoft/omsagent/run/npmdagent.sock"
-    location_control_data   "/etc/opt/microsoft/omsagent/conf/npmd_agent_config.xml"
+    omsadmin_conf_path      "%CONF_WS_DIR%/omsadmin.conf"
```

Comment:
Actually usage of "CONF_WS_DIR" is correct as that is the string being replaced by the DSC module.
I understood why it might seem wrong as CONF_DIR_WS was utilized for the default plugins but since npmd.conf is not default and the updation is not done during any addition/removal of workspace this should be perfectly fine.